### DISCUSSION TITLE
asking for support of x64 and other distribution

### DISCUSSION THREAD

**Discussion by fish4terrisa-MSDSM** [ish-app/ish#1818] [Ideas]
i386在许多发行版中已经废弃，ish会加入对x64的支持吗
Alpine的许多包是不完整的，并且有部分功能没有实现，以后ish会实现对ubuntu，redhat之类的系统的支持吗

Google translate :

i386 is deprecated in many distros, will ish add support for x64
Many packages of Alpine are incomplete, and some functions have not been implemented. Will ish implement support for systems such as ubuntu and redhat in the future?

**Comment by tower-town** [comment]
https://github.com/ish-app/ish/discussions/1845#discussioncomment-3219205
